week-1-practise/level-1/9-function-output-2.js

```diff
@@ -7,8 +7,8 @@ console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
-// - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+// - How many parameters does the function take? two
+// - What is the function name? s
+// - Where is the function called? with what parameters? the function is called on line number 5 with the following parameters('code', 'your')
+// - What does the function return? the function return s
+// - How can this function be improved? the function can be improved in my view by have a better function name and proper spacing.
```

Comment:
Better function name would definitely be an improvement! What about parameter names? :)
What would you rename the whole function to?

Reply:
I would rename the whole function the following
function learnCode(name1, name2){
return name1.concat(name2);
}
var result = learnCode('code', 'your');
 console.log(result);

Reply:
Why `learnCode`? Usually the function name should reflect what the function does - so what does it do?

Reply:
Because the function is going to log ('code', 'your'); and it is about learning how to code, well the function is concatenating the value of name1 and name2 to output it as a single string.